www/js/pages/sign-in/components/oauth/oauth.controller.js

```diff
@@ -92,11 +96,8 @@
                 result = regex.exec(_url);
                 if(result){
                     accounts.push({
-                        loginid   : result[1],
-                        token     : result[2],
-                        currency  : result[4] ? result[4] : null,
```

Comment:
Plz don't remove these 3 lines in order to prevent from `undefined` errors in the app.